### ISSUE TITLE
Website translated to RU

### ISSUE THREAD

**Issue by aristofun** [oneclick/rubyinstaller.org-website#13]
Guys professionally translated original website to russian: https://github.com/installero/rubyinstaller.ru

Could you please encourage their effort and make some users life better by writing a news post and adding the link to rubyinstaller.ru website somewhere in the footer?

Thanks

**Comment by lex111** [comment]
cc @larskanis

**Comment by larskanis** [comment]
I already asked some questions regarding the translation in #11 . I think that we need some conclusion about responsibilities and some team work to keep all translations up to date. Currently the Russian binaries are outdated and this shouldn't happen.

**Comment by stomar** [comment]
How would the integrity of provided binaries be guaranteed for a site that is not owned by rubyinstaller.org?

**Comment by aristofun** [comment]
Good point.

By preserving original links to all binaries, I guess. It is only website texts that are modified. 

I may be wrong, but strictly speaking even "owning" the .org site doesn't guarantee the binaries integrity 100% of the time. Because of the open source and public nature of the project.

**Comment by installero** [comment]
> Currently the Russian binaries are outdated

I've picked your last commits, sorry for the delay and thank's again for your work.

We'll update Russian version shortly (about a week) after yours.

**Comment by stomar** [comment]
> By preserving original links to all binaries, I guess. It is only website texts that are modified.

@aristofun That might be true right now, but could change at any time, unnoticed by visitors; note also that there is no guarantee that the GitHub repository really is the source for the website.

Don‘t misunderstand me, I do not want to imply any malicious intent; I simply do not know anything about rubyinstaller.ru and have currently no reason to trust the site. Therefore IMO it‘s an important consideration.

**Comment by aristofun** [comment]
It's a valid point of course. 
I just see no point nor reason for anybody's malicious intent in such a complicated way (by making a translation, communicating with original site owners etc.). 

Also rubyinstaller.org could technically be changed at any time, unnoticed by visitors etc. Most of your considerations are applicable to .org website as well. 
(it's just for the clarity of an argument of course, I'm sure everybody is doing a perfect job).

But I see a big point in supporting people making a valuable contribution to popularizing the Ruby.

PS: 
if you happen to personally know Ivan Nemytchenko https://twitter.com/inem or @inem, he may confirm my and @installero identity and contribution to russian ruby/rails community we've made so far. And that both of us have ever had only good intentions in mind and in our doings.

**Comment by stomar** [comment]
@aristofun Regarding your first statement: consider e.g. the infamous event-stream incident for npm. And the integrity of this project is not the point here at all (I assume the maintainers do trust themselves...)

I think it would probably be best if the translations would be incorporated into the rubyinstaller site itself.

**Comment by aristofun** [comment]
Not sure how would "npm" scenario be possible if .ru and .org website repos are okay and in sync. 

Could you please elaborate on the specifics of what kind of vulnerabilities you have in mind?

**Comment by stomar** [comment]
Please reread my very first comment; further elaboration should not be necessary.

**Comment by aristofun** [comment]
I agree with all your points except one: 

> How would the integrity of provided binaries be guaranteed for a site that is not owned by rubyinstaller.org?

How would the integrity of provided binaries be guaranteed for a rubyinstaller.org site?

Sorry, nothing personal, but it feels a little bit like discrimination here

**Comment by stomar** [comment]
Really??? Think again about the context: You are asking the maintainers of rubyinstaller.org to promote the Russian site; they need to (or at least should) consider how to make sure that the .ru site stays in sync with their site. The integrity of rubyinstaller.org is not relevant in this context at all.

**Comment by aristofun** [comment]
You're right, my ba, sorry. This way there's simply no technically reliable way to ensure integrity except trust (human factor).

**Comment by larskanis** [comment]
The RubyInstaller-3.0.0-1 files and blog post are missing on https://rubyinstaller.ru . Could you please update them?

**Comment by installero** [comment]
Hello!

Thank’s for the notice, Lars.

Will update in a couple of days.

Vadim

**Comment by installero** [comment]
Hello, Lars.

And rubyinstaller.ru is up to date.

Thank’s for your job!

Vadim
